v3/src/components/case-card/case-card-registration.ts

```diff
@@ -16,6 +18,10 @@ registerTileContentInfo({
   prefix: kCaseCardIdPrefix,
   modelClass: CaseCardModel,
   defaultContent: () => ({ type: kCaseCardTileType }),
+  getTitle: (tile) => {
+    const data = tile?.content && getTileDataSet(tile?.content)
+    return data?.title || t("DG.DocumentController.caseTableTitle")
```

Comment:
Yes. For a case table/card the title is always based on the title of the data context. Therefore, when we change a table/card title, we change the title of the data context instead of just the tile.